test/onnx/test_fx_op_consistency.py

```diff
@@ -275,6 +275,10 @@
             "Chunk", "uint8, int8, int16, float16"
         ),
     ),
+    xfail(
```

Comment:
```suggestion
    xfail_torchlib_forward_compatibility(
```

Could you rebase on main and use `xfail_torchlib_forward_compatibility` instead? Thanks!